app/views/curation_concerns/base/_representative_media.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if work.representative_id.present? && (fs = ::FileSet.load_instance_from_solr(work.representative_id)) %>
+<% if presenter.representative_id.present? && (fs = ::FileSet.load_instance_from_solr(presenter.representative_id)) %>
```

Comment:
Yes, good idea, but I don't think that's what *this* PR is really about. Can you make another ticket?